config/packages/lti1p3.yaml

```diff
@@ -28,4 +30,4 @@ lti1p3:
             platform_key_chain: "platformKey"
             tool_key_chain: null
             platform_jwks_url: null
-            tool_jwks_url: "http://localhost:8888/lti1p3/.well-known/jwks/toolSet.json"
+            tool_jwks_url: "http://simple-roster.localhost/lti1p3/.well-known/jwks/toolSet.json"
```

Comment:
I would suggest to parametrize `HOST` part of the url - it would make easier configurational steps, to just put/update in .env file, then search and update lti1p3.yaml

Reply:
Since this file may need additional adjustments according to the tool that will connect to simple roster, I don't see why do it only for the hostnames.

Reply:
Maybe also a good choice. What I'd want to avoid - configuring same stuff in N different places if for some reason you would need to switch from local to docker, from docker to test env, for example (still it would be three different places, but .env files, IMO, suits better)

Reply:
@Skyld @andreluizmachado 

Any input here to help us untie this? I still not seeing any gain, but would change if more people thinks it's better approach.

Reply:
Moved all `urls` to ENV variables. Need to move full URL and not only the host since they can change as well